gocontext.go

```diff
@@ -90,6 +90,17 @@ func StartSpan(ctx context.Context, name, spanType string) (*Span, context.Conte
 	return StartSpanOptions(ctx, name, spanType, SpanOptions{})
 }
 
+// StartExitSpan starts an exit span. If the provided context already contains
+// an exit span, this function returns a dropped span.
```

Comment:
instead of creating a new method, how about we add a `ExitSpan bool` field to `apm.SpanOptions`?